assets/js/components/StoreErrorNotices.js

```diff
@@ -25,37 +25,34 @@ import PropTypes from 'prop-types';
  * Internal dependencies
  */
 import Data from 'googlesitekit-data';
+import ErrorNotice from './ErrorNotice';
 import { STORE_NAME as CORE_MODULES } from '../googlesitekit/modules/datastore/constants';
-import { isPermissionScopeError, isInsufficientPermissionsError } from '../util/errors';
+import { isInsufficientPermissionsError } from '../util/errors';
 import { getInsufficientPermissionsErrorDescription } from '../util/insufficient-permissions-error-description';
-import ErrorText from '../components/error-text';
 const { useSelect } = Data;
 
-function StoreErrorNotice( { moduleSlug, storeName, shouldDisplayError } ) {
-	const error = useSelect( ( select ) => select( storeName ).getError() );
+export default function StoreErrorNotices( { moduleSlug, storeName, shouldDisplayError } ) {
+	const errors = useSelect( ( select ) => select( storeName ).getErrors() );
 	const module = useSelect( ( select ) => select( CORE_MODULES ).getModule( moduleSlug ) );
 
-	// Do not display if no error, or if the error is for missing scopes.
-	if ( ! error || isPermissionScopeError( error ) || ! shouldDisplayError( error ) ) {
-		return null;
-	}
-
-	let message = error.message;
-	if ( isInsufficientPermissionsError( error ) ) {
-		message = getInsufficientPermissionsErrorDescription( message, module );
-	}
-
-	return <ErrorText message={ message } reconnectURL={ error.data?.reconnectURL } />;
+	return errors
+		.map( ( error ) => {
+			if ( isInsufficientPermissionsError( error ) ) {
+				error.message = getInsufficientPermissionsErrorDescription( error.message, module );
```

Comment:
We should probably destructure this into a new object, so that we don't modify the original `error` object.
```suggestion
				error = {
					...error,
					message: getInsufficientPermissionsErrorDescription( error.message, module )
				};
```